src/Controls/tests/Xaml.UnitTests/Issues/Maui13619.xaml.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using Microsoft.Maui.ApplicationModel;
+using Microsoft.Maui.Controls.Core.UnitTests;
+using Microsoft.Maui.Graphics;
+using Microsoft.Maui.Devices;
+using NUnit.Framework;
+
+namespace Microsoft.Maui.Controls.Xaml.UnitTests
+{
+
+
+	public partial class Maui13619 : ContentPage
+	{
+		public Maui13619() => InitializeComponent();
+		public Maui13619(bool useCompiledXaml)
+		{
+			//this stub will be replaced at compile time
+		}
+
+		[TestFixture]
+		class Tests
+		{
+			[SetUp] public void Setup() => AppInfo.SetCurrent(new MockAppInfo());
+			[TearDown] public void TearDown() => AppInfo.SetCurrent(null);
+
+			[Test]
+			public void AppThemeBindingAndDynamicResource([Values(false, true)] bool useCompiledXaml)
+			{
+				var page = new Maui13619(useCompiledXaml);
+				Assert.That(page.label0.TextColor, Is.EqualTo(Colors.HotPink));
```

Comment:
I see you also went to the Barbie movie this weekend!

Reply:
I can differentiate about 9 colors, HotPink is 2 of them.